brion/morphologyPlugin.h

```diff
@@ -144,6 +141,9 @@ class MorphologyPlugin : public boost::noncopyable
     /** @copydoc brion::Morphology::flush */
     virtual void flush() = 0;
     //@}
+
+protected:
+    MorphologyInitData _data;
```

Comment:
The reason is not saving code but eliminating redundancy (DRY). The version and family where duplicated between the init data and plugin impls, with some plugins (unsurprisingly) not handling the information correctly.